components/NavigationDrawer/index.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react'
+import Image from 'next/image'
+// MUI //
+import Drawer from '@mui/material/Drawer'
+import { Box } from '@mui/system'
+import Collapse from '@mui/material/Collapse'
+// other mui dependencies
+import ExpandLess from '@mui/icons-material/ExpandLess'
+import ExpandMore from '@mui/icons-material/ExpandMore'
+// icons and text
+// import Typography from '@mui/material/Typography'
+import AccessTimeIcon from '@mui/icons-material/AccessTime'
+import SettingsIcon from '@mui/icons-material/Settings'
+import ArchiveOutlinedIcon from '@mui/icons-material/ArchiveOutlined'
+import UnarchiveOutlinedIcon from '@mui/icons-material/UnarchiveOutlined'
+import AssignmentLateOutlinedIcon from '@mui/icons-material/AssignmentLateOutlined'
+import AssignmentOutlinedIcon from '@mui/icons-material/AssignmentOutlined'
+// List component
+import ListItem from '../ListItem/'
+// List components from MUI
+import List from '@mui/material/List'
+import ListItemText from '@mui/material/ListItemText'
+import ListItemIcon from '@mui/material/ListItemIcon'
+import ListItemButton from '@mui/material/ListItemButton'
+
+export default function NavigationDrawer() {
+  // Params for the Drawer
+  const drawerWidth = 280
+  // Logo definition
+  const logo = (
+    <Image
+      src="/../public/images/CCAHT-Logo.png"
+      alt="CCAHT-logo"
+      height="70"
+      width="239"
+    />
+  )
+  // state handlers for expandable lists in drawer
+  const [open, setOpen] = React.useState(true)
+  const handleClick = () => {
+    setOpen(!open)
+  }
+  // Drawer setup
+  return (
+    <Box sx={{ display: 'flex' }}>
+      <Drawer
+        variant="permanent"
+        sx={{
+          width: drawerWidth,
+        }}
+      >
+        <Box>
+          <Box sx={{ pr: 3, mt: 5 }}>{logo}</Box>
+          <br />
+          {/* Drawer using ListItem functional component*/}
+          <ListItem icon={<AssignmentLateOutlinedIcon />} text="Dashboard" />
```

Comment:
these should function to route between pages. to do this, add a prop on the `ListItem` component you defined called `route: string`, and then pass the route for each item in to it.

inside the `ListItem` component, change the `ListItemButton` to the following:
`<ListItemButton component={Link} href={route}> ... </ListItemButton>`

Reply:
Will do, I thought this specific funtionality was for my other task on setting up the routing. Nonetheless, I need to do it.

Reply:
disregard.